### PR TITLE
Allow message color override in settings

### LINKED ISSUE #102: allow message color override in settings
I use an initial step in my pipeline to send a message about a build that has just been started

```
steps:
  - name: slack-init
    image: plugins/slack
    settings:
      webhook: https://xxxx
      channel: deploy-feed
      template: "build just started"
```

however the message color seems to depend on the build status, which at this point I guess drone assumes it being green.

https://github.com/drone-plugins/drone-slack/blob/master/plugin.go#L94

it would be nice being able to send this message as `warning`:

https://github.com/drone-plugins/drone-slack/blob/master/plugin.go#L161

I suspect the default case might never be reached.

### PR DESCRIPTION
**Pull request by wesleimp**
Close #102

### CODE REVIEW

File: plugin.go [hunk 1]
```diff
@@ -162,6 +176,15 @@ func color(build Build) string {
 	}
 }
 
+func checkAndSelectColor(color string) string {
```
Comment (bradrydzewski) [inline plugin.go]: this function should be removed.  Slack supports color labels and also supports hex values. If the user inputs an invalid value, the slack API will return an error. No need to validate this client-side.

**Comment by wesleimp** [comment]
@bradrydzewski sorry about the flags, my bad

**Comment by tboerger** [comment]
This seems to be a mix of the custom fallback template and the color pr :(

**Comment by wesleimp** [comment]
@tboerger PR #103 needs to be merged first, after that, custom fallback will desapear.

I've done that changes right on master. That's my mistake, sorry.

**Comment by wesleimp** [comment]
@tboerger Could you please review this PR again?

**Comment by tboerger** [comment]
@bradrydzewski your requested changes have been applied, right?

**Pull request merged**